features/admin.users.v1/components/user-groups-list.tsx

```diff
@@ -0,0 +1,493 @@
+                    level: AlertLevels.SUCCESS,
+                    message: t("console:manage.features.groups.notifications.updateGroup.success.message")
+                }));
+                handleUserUpdate(user.id);
+            }).catch(() => {
+                dispatch(addAlert({
+                    description: t("console:manage.features.groups.notifications.updateGroup.error.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("console:manage.features.groups.notifications.updateGroup.error.message")
+                }));
+            }).finally(() => {
+                setDeleting(false);
+                setDeletingGroup(undefined);
+                setShowDeleteConfirmationModal(false);
+            });
+    };
+
+    /**
+     * Resolves data table columns.
+     *
+     * @returns the data table columns.
+     */
+    const resolveTableColumns = (): TableColumnInterface[] => {
+        const defaultColumns: TableColumnInterface[] = [
+            {
+                allowToggleVisibility: false,
+                dataIndex: "groupName",
+                id: "groupName",
+                key: "groupName",
+                render: (group: GroupsMemberInterface): ReactNode => {
+                    const userGroup: string[] = group?.display?.split("/");
+                    const userGroupName: string = userGroup.length === 1
+                        ? userGroup[0]
+                        : userGroup[1];
+
+                    return (
+                        <Header
+                            image
+                            as="h6"
+                            className="header-with-icon"
+                            data-componentid={ `${ componentId }-item-heading` }
+                        >
+                            <UserAvatar
+                                data-componentid="users-list-item-image"
+                                name={ group.display }
+                                size="mini"
+                                image={ user.profileUrl }
+                                spaced="right"
+                                data-suppress=""
+                            />
+                            <div>
+                                { userGroupName as ReactNode }
+                            </div>
+                        </Header>
+                    );
+                },
+                title: (
+                    <div className="pl-3">
+                        { t("user:updateUser.groups.editGroups.groupList.headers.1") }
+                    </div>
+                )
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "domain",
+                id: "domain",
+                key: "domain",
+                render: (group: GroupsMemberInterface): ReactNode => {
+                    const userGroup: string[] = group?.display?.split("/");
```

Comment:
Shall we update the variable name to better match with the logic?

```suggestion
                    const groupNameSegments: string[] = group?.display?.split("/");
```

Reply:
Fixed with cb0ee2cf165319ca096aa44d358799060f08bbf9